File: Aula03/Go/Vetor04.go
// Faça um programa em golang que receba 6 numeros inteiros e mostre: ´
// Os numeros pares digitados;
// A soma dos numeros pares digitados;
// Os numeros   ımpares digitados;
// A quantidade de numeros  ımpares

package main

import (
	"fmt"
)

func main() {
	var numbers [6]int
	var sumEven int
	var countOdd int

	fmt.Println("Digite 6 números inteiros:")

	for i := range numbers {
		fmt.Printf("Número %d: ", i+1)
		fmt.Scan(&numbers[i])
	}

	fmt.Print("Números pares digitados: ")
	for _, num := range numbers {
		if num%2 == 0 {
			fmt.Print(num, " ")
			sumEven += num
		}
	}
	fmt.Println()
	fmt.Println("Soma dos números pares:", sumEven)

	fmt.Print("Números ímpares digitados: ")
	for _, num := range numbers {
		if num%2 != 0 {
			fmt.Print(num, " ")
			countOdd++
		}
	}
	fmt.Println()
	fmt.Println("Quantidade de números ímpares:", countOdd)
}

File: Aula03/Java/Vetor04.java
// faça um programa em java que receba 6 numeros inteiros e mostre:
// Os numeros pares digitados; 
// A soma dos numeros pares digitados; 
// Os numeros   Ä±mpares digitados;
// A quantidade de numeros  impares 

import java.util.Scanner;

public class Vetor04 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] numeros = new int[6];

        System.out.println("Digite 6 números inteiros:");
        for (int i = 0; i < 6; i++) {
            numeros[i] = scanner.nextInt();
        }

        mostrarNumerosPares(numeros);
        mostrarNumerosImpares(numeros);
    }

    public static void mostrarNumerosPares(int[] numeros) {
        int somaPares = 0;
        System.out.print("Números pares digitados: ");
        for (int num : numeros) {
            if (num % 2 == 0) {
                System.out.print(num + " ");
                somaPares += num;
            }
        }
        System.out.println("\nSoma dos números pares: " + somaPares);
    }

    public static void mostrarNumerosImpares(int[] numeros) {
        int quantidadeImpares = 0;
        System.out.print("Números ímpares digitados: ");
        for (int num : numeros) {
            if (num % 2 != 0) {
                System.out.print(num + " ");
                quantidadeImpares++;
            }
        }
        System.out.println("\nQuantidade de números ímpares: " + quantidadeImpares);
    }
}
